pkg/gds/store/trtl/trtl_test.go

```diff
@@ -223,6 +224,7 @@ func (s *trtlStoreTestSuite) TestDirectoryStore() {
 	require.NoError(err)
```

Comment:
On [Line 202](https://github.com/trisacrypto/directory/blob/e0be55fbb8477f63a905d6aa34079fd9f170dbd3/pkg/gds/store/trtl/trtl_test.go#L202) above and [Line 245](https://github.com/trisacrypto/directory/blob/e0be55fbb8477f63a905d6aa34079fd9f170dbd3/pkg/gds/store/trtl/trtl_test.go#L245) below, does calling `All()` release the iterator?

Reply:
Yes, `ListVASPs()` creates the iterator, and `All()` releases the iterator when it returns.

Reply:
Cool - then this looks good to go to me! Hope it sorts out CI!